chain/network/src/store/mod.rs

```diff
@@ -0,0 +1,185 @@
+#![allow(dead_code)] //TODO: remove, it will be in use in the next PR.
+/// Store module defines atomic DB operations on top of schema module.
+/// All transactions should be implemented within this module,
+/// in particular schema::StoreUpdate is not exported.
+use near_network_primitives::types::{Edge, KnownPeerState};
+use near_primitives::network::{AnnounceAccount, PeerId};
+use near_primitives::types::AccountId;
+use std::collections::HashSet;
+use tracing::debug;
+
+mod schema;
+
+pub use schema::Error;
+
+/// Store allows for performing synchronous atomic operations on the DB.
+/// In particular it doesn't implement Clone and requires &mut self for
+/// methods writing to the DB.
+pub struct Store(schema::Store);
+
+impl Store {
```

Comment:
consider calling it NetworkStore (makes it a lot easier to read - and less confusion with 'main' Store)

Reply:
NetworkStore is stuttering with the crate name. near_store::Store shouldn't be visible wherever store::Store is used (except for the moment of wrapping, but that's unavoidable).